scripts/hazard_process.py

```diff
@@ -120,6 +186,83 @@ def process_regional_hazard (country, region):
     return
 
 
+
+def convert_regional_hazard (country, region):
```

Comment:
So this one confused me. You already went from global .tif to national .shp in the national function I believe. So here, you should be going from your national .shp, into a regional .shp. Let me know if you are stuck, I can help.

Reply:
I double checked this. What confused me was that input file was .tif, but shouldn't it have been .shp?